### ISSUE TITLE
Elgg Explore user's Log after update shows a No Results found message for new and old members

### ISSUE THREAD

**Issue by TomOndiba** [Elgg/Elgg#11910]
After updating Elgg from 1.9 to Elgg 2.3, Elgg Explore user's Log e.g elgg.org/**admin/administer_utilities/logbrowser?user_guid=**  is not working as the previous elgg 1.8 and 1.9 used to work. As a result, as an admin i can not track a log of a particular user or any users logging activity on the site. When Website_Domain/admin/administer_utilities/logbrowser?user_guid is accessed by an admin, the user's log history does not show. Instead and the utilities log browser page shows **No results found** message . I also found that Elgg is no longer tracking new members or users who are registering to the site and have confirmed their emails.

~~~ Issue posted via a cellphone ~~~

**Comment by jdalsem** [comment]
This sounds unlikely as there have been many sites upgrade to 2.3 and none have reported this issue. Did you upgrade straight from 1.9 to 2.3 or did you first upgrade to 1.12 and from there upgrade to 2.3?

**Comment by iionly** [comment]
It seems to work without any issues for me (Elgg 2.3.6, php 5.3.35). @TomOndiba do you happen to have the "No Logging" plugin installed? This would explain at least why the logbrowser shows no log entries (as there wouldn't be any entries created in the first place). Still, the listing of new members wouldn't be affected by the "No logging" plugin. Maybe something has gone wrong during the upgrade, e.g. some files missing/incomplete or some deprecated files from the former Elgg version that are no longer part of Elgg 2.3 remaining. Suggestion would be to delete all core files (keeping settings.php and .htaccess) and then copy the latest release again into the install folder. Maybe it works then (after flushing the cache/running upgrade script again).

**Comment by TomOndiba** [comment]
@jdalsem, I upgraded Elgg versions one at a time ( Elgg 1.9 then to 1.10 to  1.11, 1.12, then finally 2.3.). Everything seemed to work normally immediately after upgrade, but a few weeks later i realized that when new members were signing up or registering to the site, the log browser utility did not show any log events for new users. A logbrowser plugin that allows the admin to browse the system event log has always been active.

**Comment by TomOndiba** [comment]
@iionly , I don't have "No Logging" plugin installed. The only plugin that is active besides **logbrowser** plugin is a **logrotate**. You might be right. Perhaps during upgrade from ( Elgg 1.9 then to 1.10 to 1.11, 1.12, then finally 2.3.) something might have gone wrong. In a few days, I am planning to upgrade again from 2.3 to 2.3.6 and see if the same problem persists.

The only reason why i realized that this is not an isolated issue is because a [user on Elgg community](https://elgg.org/comment/view/2761472/2756230) mentioned the same problem.

**Comment by jeabakker** [comment]
@TomOndiba after each upgrade did you also check if there were any upgrades yet to run on the site? 
I can think of one upgrade in 2.2 (which will also be shown in 2.3) which updates some columns in various tables.

I also checked the elgg.org website and in the logbrowser i still see new activity present.

Are you able to verify that no new rows are created in the database? Maybe it's just a display error?

**Comment by TomOndiba** [comment]
@jeabakker , I can remember seeing some errors while i was upgrading elgg1.12 to 2.2. However, the upgrade wet on after i deleted the elgg 2.2 and reinstalled again. Here are some of the errors that i were logged in my elgg_error_log
`[03-Jul-2017 15:35:52 America/Detroit] PHP Parse error:  syntax error, unexpected 'class' (T_CLASS), expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$' in /home/tm/public_html/home.com/vendor/elgg/elgg/engine/classes/Elgg/Di/ServiceProvider.php on line 112
[03-Jul-2017 15:47:50 America/Detroit] PHP Parse error:  syntax error, unexpected 'class' (T_CLASS), expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$' in /home/tm/public_html/home.com/vendor/elgg/elgg/engine/classes/Elgg/Di/ServiceProvider.php on line 126 `

**Comment by jeabakker** [comment]
@TomOndiba the error 
```
[03-Jul-2017 15:35:52 America/Detroit] PHP Parse error: syntax error, unexpected 'class' (T_CLASS), expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$' in /home/tm/public_html/home.com/vendor/elgg/elgg/engine/classes/Elgg/Di/ServiceProvider.php on line 112
```
Is related to not using PHP 5.5+, but you fixed this yourself.

For Elgg 2.3 the PHP requirement is 5.6+, Have you checked that?

and could tell me 

> Are you able to verify that no new rows are created in the database? Maybe it's just a display error?

**Comment by TomOndiba** [comment]
@jeanbakker, Thanks for your reply and concern. I am using elgg 2.3 and  PHP Version 5.6.30.  Do you know new rows that were supposed to be crated in the database? If I know the row that were changed or added, i might add them manually and see if the log details will start logging new entries for new users.

**Comment by iionly** [comment]
@TomOndiba I think @jeabakker refers to the log table in the database with regards to new rows getting created there. They should get created automatically (don't think it wise to create anything manually there). If something is wrong with the log table preventing new log entries/rows getting created I would assume that the server error log would contain some entries about it.

**Comment by merilturock** [comment]
I have done my upgrades properly from each version vise .. I am sure my log browser was working in elgg 2.3.5 after upgrading to elgg 2.3.6 only i am facing this problem my PHP version is 5.6.36

**Comment by jeabakker** [comment]
Here is a compare of what changed between 2.3.5 and 2.3.6 https://github.com/Elgg/Elgg/compare/2.3.5...2.3.6

I can't find anything related to the system log.

As i mentioned [before ](https://github.com/Elgg/Elgg/issues/11910#issuecomment-382772509) I still see new log entries being created on some of the Elgg 2.3.6 sites i manage

**Comment by merilturock** [comment]
The log browser issue got solved in elgg 2.3.8 for me thank you team for your efforts ..

**Comment by jdalsem** [comment]
Closing due to lack of news
